test/CMakeLists.txt

```diff
@@ -208,7 +212,7 @@ else ()
   add_test(NAME test_ess COMMAND mcmc_diagnostics_test -tc=ess)
   add_test(NAME test_geweke COMMAND mcmc_diagnostics_test -tc=geweke)
   add_test(NAME test_raftery COMMAND mcmc_diagnostics_test -tc=raftery)
-  
```

Comment:
please remove trailing whitespace, and fix indentation in this file

Reply:
I am changing the indentation so that it is close to the original file.

Reply:
The new line is not an issue. The trailing whitespace is. 
```
Trailing whitespace is any spaces or tabs after the last non-whitespace character on the line until the newline.
```
Those are visible in the diff.

Reply:
Done